### ISSUE TITLE
Joints limits in ar.ros2_control.xacro mismatch w/ AR4_MK3.ino

### ISSUE THREAD

**Issue by navdotnetreqs** [ycheng517/ar4_ros_driver#18]
Hi,

for clarity's sake I opened a new issue of this.

I noticed that the joint limits defined in ar.ros2_control.xacro don't match the ones in AR4_MK3.ino - they seem to be the values for the MK2.

Eg.

```
<joint name="joint_6">
        <command_interface name="position">
          <param name="min">{**-155***pi/180}</param>
          <param name="max">{**155***pi/180}</param>
        </command_interface>
        <state_interface name="position">
          <param name="initial_value">0.0</param>
        </state_interface>
        <param name="position_offset">${joint_offset_parameters['joint_6']}</param>
      </joint>
```

**Comment by ycheng517** [comment]
Apologies for the wait on addressing this issue, but this should be fixed with https://github.com/ycheng517/ar4_ros_driver/pull/20
